api/gin/webServer.go

```diff
@@ -202,6 +207,25 @@ func (ws *webServer) UpdateFacade(facade shared.FacadeHandler) error {
 	return nil
 }
 
+func (ws *webServer) setOptionsForClientIP(engine *gin.Engine) error {
+	engine.ForwardedByClientIP = ws.config.GeneralConfig.Gin.ForwardedByClientIP
+
+	engine.TrustedPlatform = ws.config.GeneralConfig.Gin.TrustedPlatform
+	engine.RemoteIPHeaders = ws.config.GeneralConfig.Gin.RemoteIPHeaders
+
+	trustedProxies := ws.config.GeneralConfig.Gin.TrustedProxies
+	if len(trustedProxies) == 0 {
+		// disabled trusted proxies check
+		// will get IP directly from `RemoteAddr`, since header are not trustworthy
+		err := engine.SetTrustedProxies(nil)
```

Comment:
```suggestion
		return engine.SetTrustedProxies(nil)
```
to avoid second call

Reply:
changed